tests/perf-test/ReportActionsList.perf-test.js

```diff
@@ -0,0 +1,191 @@
+import {measurePerformance} from 'reassure';
+import Onyx from 'react-native-onyx';
+import {screen, fireEvent} from '@testing-library/react-native';
+import ReportActionsList from '../../src/pages/home/report/ReportActionsList';
+import ComposeProviders from '../../src/components/ComposeProviders';
+import OnyxProvider from '../../src/components/OnyxProvider';
+import {ReportAttachmentsProvider} from '../../src/pages/home/report/ReportAttachmentsContext';
+import {WindowDimensionsProvider} from '../../src/components/withWindowDimensions';
+import {LocaleContextProvider} from '../../src/components/LocaleContextProvider';
+import * as LHNTestUtils from '../utils/LHNTestUtils';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatchedUpdates';
+import PusherHelper from '../utils/PusherHelper';
+import variables from '../../src/styles/variables';
+import {ActionListContext, ReactionListContext} from '../../src/pages/home/ReportScreenContext';
+
+const ONYXKEYS = {
```

Comment:
you can use the globally available ONYXKEYS constant (less error prone) with `import ONYXKEYS from '../../src/ONYXKEYS';`

Reply:
i remember missing a single plural form here and spending hours debugging it :P